knexfile.js

```diff
@@ -24,5 +32,15 @@ module.exports = {
     seeds: {
       directory: "./database/seeds"
     }
+  },
+  production: {
+    client: "pg",
+    connection: prodDbConnection,
```

Comment:
I still need to finish going through Luis' vid explaining this portion of it, but in my old project's knexfile for pg migration I had `    connection: prodDbConnection + "?ssl=true",` on this portion, may or may not be crucial